### ISSUE TITLE
Update contributor's guide to reflect branch changes

### ISSUE THREAD

**Issue by Hanziness** [Hanziness/AnotherPomodoro#134]
Branching strategies have changed and this should be reflected in the contributor's guide:

### `develop` is the new default branch
Just like before, all pull requests should be made against this branch and it will continue to contain the latest (not necessarily production-ready) version of the application. It is only possible to merge into this branch through pull requests.

**Reason of change**: GitHub's issue and pull request linking allows automatically closing relevant issues once a pull request is merged *into the default branch*. Since pull requests were made against a non-default branch, issues had to be manually closed. This change should make automation a lot easier.

### Deployments are made from `prod` and `preview`
There is a new `prod` branch that serves the production-ready version of the application (the same thing that `master` was doing). Netlify will auto-deploy this branch to the default domain. The `preview` branch has not changed, it can be used to host pre-release versions of the app.

### `master` is no more
`prod` having taken the role of hosting the production version and `develop` becoming the default branch, the `master` branch has been deleted.
